pkg/logging/fields.go

```diff
@@ -68,6 +70,16 @@ type astNodeField struct {
 	content string
 }
 
+type entryMessage struct{}
+
+func (field entryMessage) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	return nil
+}
+
+func SendEntryMessage() zap.Field {
```

Comment:
Can you add some godoc for this explaining what it does / when to use it?

Also, this could be simplified I think to just
```go
var SendEntryMessage = zap.Object("entryMessage", struct{}{})
```
(and checking `f == SendEntryMessage` in the listener instead of the Key)

Reply:
yep will do 👍